### DISCUSSION TITLE
Is Github Student pack renewable?

### DISCUSSION THREAD

**Discussion by realwenura** [community/community#46408] [GitHub Education] [answered]
Is Github Student pack renewable? or when renewing we have access to the free services again?

**Comment by jge162** [comment]
Yes, you just need to provide current info to renew your student pack. @wenuraonline

**Comment by NHbinaaa111** [comment, marked as answer]
Once your GitHub Global Campus access expires, you may reapply if you're still eligible, although some of our partner offers for GitHub Student Developer Pack cannot renew. Most of the timed offers from our partners start once you set them up.

-Official answer from GitHub docs

**Reply by realwenura** [reply]
Domains?

**Reply by jge162** [reply]
Domains? For what.

**Reply by jge162** [reply]
@wenuraonline here is link to [Education support](https://support.github.com/request/education?tags=hubberfy_education). Not sure what you mean by domains.

**Reply by apoorvkapsime** [reply]
What services are not renewable

**Reply by myselfprincee** [reply]
Hey, I want to know that which are the Offers that are non-renewable ??

**Comment by cankush625** [comment]
@wenuraonline GitHub student pack is renewable. You need to provide your current valid information such as your college ID card or equivalent docs. Also, you might need an active email ID provided by your college.
Yes. After renewing the student developer pack, you will get access to all of the free services that are offered by that pack.
